### ISSUE TITLE
TRAINING CODE

### ISSUE THREAD

**Issue by doudoublues** [zhaominyiz/RFDA-PyTorch#2]
hello，I'm interested in your great work，could you release the training code so that i can reproduce the result for my thesis. THX!

**Comment by zhaominyiz** [comment]
Hi,
In my scheduler, i plan to clean and release the training code in Feb.2022. 
However, if you are robust enough to my messy implementation, or is in emergency need for academic research, you may post me an email: zhaomy20@fudan.edu.cn. I will share them in next week

**Comment by doudoublues** [comment]
thx. ！

**Comment by zhaominyiz** [comment]
> thx. ！

Wish you good luck, if visualization video clips are needed, feel free to contact me.

**Comment by zhaominyiz** [comment]
They are uploaded
